styles.css

```diff
@@ -582,3 +582,46 @@ textarea:focus {
   border-radius: 8px;
   color: #5e6c84;
 }
+
+.nav-style {
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  left: 0;
+  top: 0;
+  z-index: 10;
+}
+
+.nav-style ul {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-top: 50px;
+  list-style: none;
+}
```

Comment:
- [ ] Please make your mobile background blurry to look like the given design. You can use [backdrop-filter](https://www.w3schools.com/cssref/css3_pr_backdrop-filter.asp) property for this.
- [ ] Please add the correct margin values between mobile menu links to match the given design. 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/195064044-f83c9918-94e7-46a7-8745-1a8d947cad51.png)  | ![image](https://user-images.githubusercontent.com/68796232/195063994-555c615e-cfcf-4a6e-b8fa-4627346055bb.png)  |